satellitevu/apis/otm.py

```diff
@@ -50,3 +50,95 @@ def get_feasibility(self, *, id: Union[UUID, str]):
             method="GET",
             url=self.url(f"/tasking/feasibilities/{str(id)}"),
         )
+
+    def get_feasibility_response(self, *, id: Union[UUID, str]):
+        """
+        TODO: Docs
+        """
+        return self.make_request(
+            method="GET",
+            url=self.url(f"/tasking/feasibilities/{str(id)}/response"),
+        )
+
+    def list_feasibility_requests(
+        self, *, per_page: int = 25, page_token: Optional[str] = None
+    ):
+        """
+        TODO: Docs
+        """
+        query = f"per_page={per_page}"
+        if page_token:
+            query += f"&token={page_token}"
+
+        return self.make_request(
+            method="GET", url=self.url(f"/tasking/feasibilities/?{query}")
+        )
+
+    def post_order(
+        self,
+        *,
+        coordinates: Union[Tuple[float, float], Tuple[float, float, float]],
```

Comment:
Currently this can only take point geometries. Ideally we'd want to give it any geometry so maybe  mimicking what the archive service would be more consistent with our apis as well? 

https://github.com/SatelliteVu/satellitevu-client-python/blob/main/satellitevu/apis/archive.py

Reply:
sorry, got confused with the search endpoint! 🤦‍♀️